src/main/java/org/generationcp/middleware/operation/saver/ExperimentPropertySaver.java

```diff
@@ -31,11 +30,11 @@
 
 public class ExperimentPropertySaver extends Saver {
 
-	private StringBuilder batchExperimentPropInsertSql = new StringBuilder();
+	private final StringBuilder batchExperimentPropInsertSql = new StringBuilder();
```

Comment:
we can remove this field and probably rename `saveInBatchOrUpdateProperty` to just `saveOrUpdateProperty`